packages/api/test/e2e/cennzx.e2e.ts

```diff
@@ -87,6 +129,18 @@ describe('CENNZX e2e queries/transactions', () => {
         );
         done();
       });
+
+      describe('feeExchange derive queries with negative flow', () => {
```

Comment:
Not a big deal, but this test is not part of `Negative flow with no liquidity in pool`, could get it in the same level of `describe('Negative flow with no liquidity in pool')`?

Reply:
It is part of negative flow with no liquidity in pool.. This derive query will result in error, only when there is no enough liquidity in the pool

Reply:
Gotcha 👍